xsd/netex_part_1/part1_ifopt/netex_ifopt_equipmentSigns_version.xsd

```diff
@@ -158,6 +158,21 @@ Rail transport, Roads and Road transport
 					<xsd:documentation>whether SIGN is machine readable.</xsd:documentation>
 				</xsd:annotation>
 			</xsd:element>
+			<xsd:element name="AudioAnnoucementType" type="AudioAnnoucementTypeEnumeration" minOccurs="0">
+				<xsd:annotation>
+					<xsd:documentation>Indicate whether the display may be read by the audio tool .</xsd:documentation>
+				</xsd:annotation>
+			</xsd:element>
+			<xsd:element name="FontSize" type="FontSizeEnumeration" minOccurs="0">
+				<xsd:annotation>
+					<xsd:documentation>Character font size .</xsd:documentation>
+				</xsd:annotation>
+			</xsd:element>
```

Comment:
@nick-knowles in your version FontSize is not present. Please advise.

Reply:
I confirm that FontSize is important